react-native/react/common-adapters/button.desktop.js

```diff
@@ -0,0 +1,51 @@
+/* @flow */
+
+import React, {Component} from '../base-react'
+import {FlatButton} from 'material-ui'
+import {globalStyles, globalColors} from '../styles/style-guide'
+
+export default class Button extends Component {
+  render () {
+    const rootStyle = this.props.primary ? styles.buttonPrimary : styles.buttonSeconary
```

Comment:
:sheep: 
Since you have types defined for this you can do 
```js
import type {Props} from './button'
```

Then here add 
`props: Props`

This will help the `Props` type not get out of sync with what this component uses